handbook/engineering/search/index.md

```diff
@@ -47,15 +50,20 @@ Our current focus is documented in the [tracking issue for the current milestone
   - **Outcomes:** OpenGrok users eagerly migrate to Sourcegraph, and can run complex searches on Sourcegraph.
   - **Owner:** Rijnard
   - **Status:** In progress
+  - **Estimated completion:** TODO
```

Comment:
@rvantonder could you add an (even rough) estimated completion milestone for this work?